Dynatrace/dynatrace-operator
# Change release workflow to release all architectures to rhcc

## Description

Ticket: https://dt-rnd.atlassian.net/browse/K8S-10606

Currently we only push the -amd64 image to rhcc, but instead we should now push all architectures to rhcc and also create a manifest (with all architectures) like we do it for all other platforms. 

## How can this be tested?

Do a prerelease or a release (😄 ) and check if the manifest is correctly pushed to rhcc, aswell as the openshift.yaml and openshift-csi.yaml have the correct image (registry.connect.redhat) set
